pages/studentPortal/quizcomponent/edu-background.tsx

```diff
@@ -1,3 +1,49 @@
+<<<<<<< HEAD
+import { React, useState } from "react";
```

Comment:
move edu-background into a folder in components. I stored mine in components/quizzes, but ive not merged in my PR yet so it is not on main.